[2022-05-15 03:37:48,584] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: dbt_advanced_dag_utility.dbt_test.test.jaffle_shop.stg_jaffle_shop__orders scheduled__2022-05-14T03:30:38.730607+00:00 [queued]>
[2022-05-15 03:37:48,644] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: dbt_advanced_dag_utility.dbt_test.test.jaffle_shop.stg_jaffle_shop__orders scheduled__2022-05-14T03:30:38.730607+00:00 [queued]>
[2022-05-15 03:37:48,650] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2022-05-15 03:37:48,655] {taskinstance.py:1242} INFO - Starting attempt 2 of 2
[2022-05-15 03:37:48,659] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2022-05-15 03:37:48,733] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): dbt_test.test.jaffle_shop.stg_jaffle_shop__orders> on 2022-05-14 03:30:38.730607+00:00
[2022-05-15 03:37:48,750] {standard_task_runner.py:52} INFO - Started process 5686 to run task
[2022-05-15 03:37:48,771] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'dbt_advanced_dag_utility', 'dbt_test.test.jaffle_shop.stg_jaffle_shop__orders', 'scheduled__2022-05-14T03:30:38.730607+00:00', '--job-id', '109', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpij5a1z0l', '--error-file', '/tmp/tmpl3y2g60q']
[2022-05-15 03:37:48,774] {standard_task_runner.py:77} INFO - Job 109: Subtask dbt_test.test.jaffle_shop.stg_jaffle_shop__orders
[2022-05-15 03:37:48,948] {logging_mixin.py:109} INFO - Running <TaskInstance: dbt_advanced_dag_utility.dbt_test.test.jaffle_shop.stg_jaffle_shop__orders scheduled__2022-05-14T03:30:38.730607+00:00 [running]> on host b0833397c3cf
[2022-05-15 03:37:49,090] {taskinstance.py:1429} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dbt_advanced_dag_utility
AIRFLOW_CTX_TASK_ID=dbt_test.test.jaffle_shop.stg_jaffle_shop__orders
AIRFLOW_CTX_EXECUTION_DATE=2022-05-14T03:30:38.730607+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-14T03:30:38.730607+00:00
[2022-05-15 03:37:49,092] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-05-15 03:37:49,095] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'dbt --no-write-json test --target dev --models stg_jaffle_shop__orders --profiles-dir /opt/***/dags/jaffle_shop --project-dir /opt/***/dags/jaffle_shop']
[2022-05-15 03:37:49,160] {subprocess.py:85} INFO - Output:
[2022-05-15 03:37:56,273] {subprocess.py:89} INFO - 03:37:56  Running with dbt=1.1.0
[2022-05-15 03:37:56,491] {subprocess.py:89} INFO - 03:37:56  Found 5 models, 4 tests, 0 snapshots, 0 analyses, 170 macros, 0 operations, 0 seed files, 3 sources, 0 exposures, 0 metrics
[2022-05-15 03:37:56,502] {subprocess.py:89} INFO - 03:37:56
[2022-05-15 03:37:56,522] {subprocess.py:89} INFO - 03:37:56  [[33mWARNING[0m]: Nothing to do. Try checking your model configs and model specification args
[2022-05-15 03:37:58,740] {subprocess.py:93} INFO - Command exited with return code 0
[2022-05-15 03:37:58,817] {taskinstance.py:1280} INFO - Marking task as SUCCESS. dag_id=dbt_advanced_dag_utility, task_id=dbt_test.test.jaffle_shop.stg_jaffle_shop__orders, execution_date=20220514T033038, start_date=20220515T033748, end_date=20220515T033758
[2022-05-15 03:37:58,918] {local_task_job.py:212} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-05-15 03:37:58,927] {process_utils.py:100} INFO - Sending Signals.SIGTERM to GPID 5686
[2022-05-15 03:37:58,961] {process_utils.py:66} INFO - Process psutil.Process(pid=5686, status='terminated', exitcode=0, started='03:37:48') (5686) terminated with exit code 0
